### DISCUSSION TITLE
Installing new libraries

### DISCUSSION THREAD

**Discussion by lakshmid13579** [code-kern-ai/refinery#179] [Q&A]
How to install a new library like fuzzy-wuzzy to work within refinery?

**Comment by jhoetter** [comment]
Hi @LakshmiPanguluri, sorry for the delayed response. We're currently preparing another open-source library with tons of heuristics that you can copy paste, which will include the fuzzywuzzy library (here's a sneak peak: https://bricks.kern.ai/home). Meaning in ca two weeks, fuzzywuzzy will be in refinery anyways.

Pinging @FelixKirsch: Is there an easy approach such that developers like LakshmiPanguluri can install their own libraries without having to build the services from scratch?

**Reply by FelixKirsch** [reply]
No, the best approach to installing your own libraries is to build the execution environments yourself with the added requirements. If that's interesting, I can write down a short tutorial on how to do that.
